en/docs/learn/enterprise-integration-patterns/messaging-channels/point-to-point-channel.md

```diff
@@ -1,132 +1,169 @@
 # Point-to-Point Channel
 
-The Point-to-Point Channel EIP pattern allows only a single receiver to consume a sent message when there are multiple receivers waiting to consume it.
+This page explains how you can implement a sample scenario of Point-to-Point Channel EIP using WSO2 Micro Integrator.
+
+## Introduction to Point-to-Point Channel
+
+The Point-to-Point Channel EIP allows only a single receiver to consume a sent message when multiple receivers are waiting to consume it.
 
 !!! info
     For more information, go to [Point-to-Point Channel](http://www.eaipatterns.com/PointToPointChannel.html). 
 
 ## Sample scenario
 
-The sample scenario is an inventory for stocks. It illustrates how a stock quote is generated, which only a single consumer receives at a given time. The diagram below depicts how to simulate the sample scenario using the ESB profile.
+The sample scenario is an inventory for stocks. It illustrates how a stock quote is generated, which only a single consumer receives at a given time. The diagram below depicts how to simulate the sample scenario.
 
 ![Point-to-Point Channel]({{base_path}}/assets/img/learn/enterprise-integration-patterns/messaging-channels/point-to-point.png)
 
 ## Synapse configurations of the artifacts
 
-When you unzip the ZIP file you download below in Step 6 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/synapse-config` directory. For more information about these artifacts, go to WSO2 EI Documentation.
-
-**Proxy Service**
-
-```
-<proxy xmlns="http://ws.apache.org/ns/synapse" name="PointToPointProxy" transports="http https" startOnLoad="true" >
-       <target>
-           <inSequence>
-               <send>
-                   <endpoint>
-                       <!-- Channel With Multiple Endpoints Load Balancer Will Ensure that only one will receive it -->
-                       <loadbalance>
-                           <endpoint>
-                               <address uri="http://localhost:9000/services/SimpleStockQuoteService/" />
-                           </endpoint>
-                           <endpoint>
-                               <address uri="http://localhost:9001/services/SimpleStockQuoteService/" />
-                           </endpoint>
-                           <endpoint>
-                               <address uri="http://localhost:9002/services/SimpleStockQuoteService/" />
-                           </endpoint>
-                       </loadbalance>
-                   </endpoint>
-               </send>
-           </inSequence>
-           <outSequence>
-               <respond/>
-           </outSequence>
-       </target>
-</proxy>
-```
+When you unzip the ZIP file you downloaded below in step 9 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/wso2mi/artifacts` directory. For more information about these artifacts, go to WSO2 MI Documentation.
```

Comment:
'For more information about these artifacts, go to WSO2 MI Documentation.' Let's point to synapse artifacts page, as we are already in MI docs

Reply:
let's point to https://mi.docs.wso2.com/en/latest/develop/intro-integration-development/